### ISSUE TITLE
[insiders] Activating extension 'vscode.search-result' failed

### ISSUE THREAD

**Issue by DanTup** [microsoft/vscode#85534]
When using Insiders today and running the mock-debug extension in the extension host, I see this message pop up (I do not see it in the main Insiders window, only when running the Mock Debug extension in the extension host):

> Activating extension 'vscode.search-result' failed: Cannot find module '/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/search-result/out/extension.js' Require stack: - /Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/loader.js - /Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/bootstrap-amd.js - /Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/bootstrap-fork.js.
onDidNotificationChange @ /Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:

**Comment by weinand** [comment]
@DanTup I cannot reproduce but most likely because I don't really understand the steps I have to do...
What I did:
- in a VS Code insiders window I opened the latest version of vscode-mock-debug
- pressed F5
- in the resulting Window i opened a simple mock test project with a readme.md file and a launch.json
- pressed F5

Observe: mock debugging readme.md worked fine; no search-result message popping up

Are these the correct steps?

**Comment by DanTup** [comment]
@weinand sorry, I should've been more specific. I don't think this is related to MockDebug, I just happened to be using it.

Here's exactly what I'm doing

- Open VS Code Insiders
- Open a clean clone of mock-debug
- Press F5 to launch the extension host with mock-debug running in it
- The error appears

It literally appears at startup of the extension host without me doing anything.

I don't know if it's related (since I don't think I saw it yesterday), but as well as this error appearing in the extension host, back in the main VS Code window (where I have the mock-debug source open) I see this error appear too:

<img width="544" alt="Screenshot 2019-11-26 at 2 58 13 pm" src="https://user-images.githubusercontent.com/1078012/69644942-42fb7480-105d-11ea-9986-6acaf37137bd.png">

And this is the Chrome dev tools console for the main instance:

<img width="812" alt="Screenshot 2019-11-26 at 2 59 11 pm" src="https://user-images.githubusercontent.com/1078012/69645007-5a3a6200-105d-11ea-8564-98b85bb8d796.png">
